---
float32[6] current